Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs

```diff
@@ -1,26 +1,73 @@
-﻿using Microsoft.SfB.PlatformService.SDK.Common;
+using Microsoft.SfB.PlatformService.SDK.Common;
```

Comment:
ConsoleLogger should not log to file. If you want, create a new class FileLogger and use that in samples.